pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -37,12 +38,15 @@ Once the build configuration is created you may need to run a build with 'start-
   // Create a build config from a remote repository using its beta2 branch
   $ %[1]s new-build https://github.com/openshift/ruby-hello-world#beta2
 
+  // Create a build config using a Dockerfile specified as an argument
+  $ %[1]s new-build -D $'FROM centos7\nRUN yum install apache'
```

Comment:
can I specify the Dockerfile filename? Or I have to do `oc new-build -D $'$(cat Dockerfile)'` ?